week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -11,12 +11,15 @@ We made a start on this for you here:
 let bottle = {
   volume: 0,
   fill: function() {
+    this.volume = 100;
     // calling this function should make you bottles volume = 100;
   },
   drink: function() {
+    this.volume = this.volume - 10;
     // calling this function should decrease your bottles volume by 10;
   },
   empty: function() {
+    return this.volume === 0 ? true : null;

```

Comment:
Well done for getting used to ternary operators Ola!
`this.volume === 0` expression returns a boolean value `true` or `false`
Why not just return that in this function?